coala_quickstart/Extensions.py

```diff
@@ -1,166 +1,166 @@
+        '.matlab': ['Matlab', 'Octave'],
+        '.m': ['Objective-C', 'Matlab'],
+        '.mm': ['Objective-C'],
+        '.cl': ['OpenCL'],
+        '.opencl': ['OpenCL'],
+        '.aw': ['PHP'],
+        '.ctp': ['PHP'],
+        '.php': ['PHP'],
+        '.php3': ['PHP'],
+        '.php4': ['PHP'],
+        '.php5': ['PHP'],
+        '.phpt': ['PHP'],
+        '.nqp': ['Perl'],
+        '.perl': ['Perl'],
+        '.ph': ['Perl'],
+        '.pl': ['Perl'],
+        '.plx': ['Perl'],
+        '.pm6': ['Perl'],
+        '.pod': ['Perl'],
+        '.psgi': ['Perl'],
+        '.gyp': ['Python'],
+        '.py': ['Python'],
+        '.pyt': ['Python'],
+        '.pyw': ['Python'],
+        '.wsgi': ['Python'],
+        '.xpy': ['Python'],
+        '.r': ['R'],
+        '.builder': ['Ruby'],
+        '.gemspec': ['Ruby'],
+        '.god': ['Ruby'],
+        '.irbrc': ['Ruby'],
+        '.mspec': ['Ruby'],
+        '.podspec': ['Ruby'],
+        '.rb': ['Ruby'],
+        '.rbuild': ['Ruby'],
+        '.rbw': ['Ruby'],
+        '.rbx': ['Ruby'],
+        '.ru': ['Ruby'],
+        '.thor': ['Ruby'],
+        '.watchr': ['Ruby'],
+        '.scss': ['SCSS'],
+        '.sql': ['SQL'],
+        '.scl': ['Scala'],
+        '.scala': ['Scala'],
+        '.aux': ['TeX'],
+        '.bib': ['TeX'],
+        '.dtx': ['TeX'],
+        '.ins': ['TeX'],
+        '.ltx': ['TeX'],
+        '.mkii': ['TeX'],
+        '.mkiv': ['TeX'],
+        '.mkvi': ['TeX'],
+        '.sty': ['TeX'],
+        '.tex': ['TeX'],
+        '.toc': ['TeX'],
+        '.ts': ['TypeScript'],
+        '.vhdl': ['VHDL'],
+        '.v': ['Verilog'],
+        '.veo': ['Verilog'],
+        '.axml': ['XML'],
+        '.ccxml': ['XML'],
+        '.clixml': ['XML'],
+        '.cproject': ['XML'],
+        '.dita': ['XML'],
+        '.ditamap': ['XML'],
+        '.ditaval': ['XML'],
+        '.glade': ['XML'],
+        '.grxml': ['XML'],
+        '.jelly': ['XML'],
+        '.kml': ['XML'],
+        '.mxml': ['XML'],
+        '.plist': ['XML'],
+        '.pluginspec': ['XML'],
+        '.ps1xml': ['XML'],
+        '.psc1': ['XML'],
+        '.pt': ['XML'],
+        '.rdf': ['XML'],
+        '.rss': ['XML'],
+        '.scxml': ['XML'],
+        '.svg': ['XML'],
+        '.tmcommand': ['XML'],
+        '.tml': ['XML'],
+        '.tmlanguage': ['XML'],
+        '.tmpreferences': ['XML'],
+        '.tmsnippet': ['XML'],
+        '.tmtheme': ['XML'],
+        '.ui': ['XML'],
+        '.vxml': ['XML'],
+        '.wsdl': ['XML'],
+        '.wxi': ['XML'],
+        '.wxl': ['XML'],
+        '.wxs': ['XML'],
+        '.x3d': ['XML'],
+        '.xaml': ['XML'],
+        '.xlf': ['XML'],
+        '.xliff': ['XML'],
+        '.xmi': ['XML'],
+        '.xml': ['XML'],
+        '.xsd': ['XML'],
+        '.xul': ['XML'],
+        '.zcml': ['XML'],
+        '.reek': ['YAML'],
+        '.yaml': ['YAML'],
+        '.yml': ['YAML'],
+        '.rest': ['reStructuredText'],
+        '.rst': ['reStructuredText']}
```

Comment:
please use sets, not tuples

Sincerely,

Lasse Schuirmann

lasse@schuirmann.net
http://coala-analyzer.org/ - http://viperdev.io/ - http://gitmate.com/

On 27 June 2016 at 18:05, Zatreanu Adrian-Gabriel <notifications@github.com>
wrote:

> In coala_quickstart/Extensions.py
> <https://github.com/coala-analyzer/coala-quickstart/pull/20#discussion_r68604314>
> :
>
> > +        '.wxl': ['XML'],
> > +        '.wxs': ['XML'],
> > +        '.x3d': ['XML'],
> > +        '.xaml': ['XML'],
> > +        '.xlf': ['XML'],
> > +        '.xliff': ['XML'],
> > +        '.xmi': ['XML'],
> > +        '.xml': ['XML'],
> > +        '.xsd': ['XML'],
> > +        '.xul': ['XML'],
> > +        '.zcml': ['XML'],
> > +        '.reek': ['YAML'],
> > +        '.yaml': ['YAML'],
> > +        '.yml': ['YAML'],
> > +        '.rest': ['reStructuredText'],
> > +        '.rst': ['reStructuredText']}
>
> agreed, should be tuples, they are immutable
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/coala-analyzer/coala-quickstart/pull/20/files/87a1cc17d38794d391ab024a4abc235ad82a8dc5..33e97d3c90103fb497b92659d40ddba09ac5269d#r68604314>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AFc6KM1pZFT0fugL0HB3FPMrb9fFjY98ks5qP_TGgaJpZM4IzlPV>
> .
>